omod/src/main/resources/config.xml

```diff
@@ -74,7 +74,8 @@
 
     <filter-mapping>
         <filter-name>${project.parent.artifactId}RequireLoginLocationFilter</filter-name>
-        <url-pattern>/*</url-pattern>
+        <url-pattern>/</url-pattern>
+        <url-pattern>*.page</url-pattern>
```

Comment:
@icrc-fdeniger   thanks working on this,i am also wondering whether /* pattern doesn't taken your newly introduced patterns?

Reply:
Before my changes:
the filter was applied on all the resources and the filter was refined in the java code to only be applied on .page.
So I changed it to only _do the filter_ in the filter-mapping